website/docs/advanced-concepts/granular-access-control/how-to-guides/configure-default-permissions.mdx

```diff
@@ -0,0 +1,164 @@
+---
+description: >-
+  Set up default access to apps in Granular Access Control
+title: Configure Default Access to Apps
+hide_title: true
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<!-- vale off -->
+
+<div className="tag-wrapper">
+ <h1>Configure Default Access to Apps</h1>
+
+<Tags
+tags={[
+{ name: "Business", link: "https://www.appsmith.com/pricing", additionalClass: "business" }
+]}
+/>
+
+</div>
+
+<!-- vale on -->
+
+To simplify access management and ensure consistency across the Appsmith instance, you may need to provide default access to all users. This page explains how to provide default access to all users accessing the Appsmith instance.
+
+## Prerequisites
+
+Before you start, ensure you have:
+
+- A self-hosted Appsmith instance running the Commercial Edition. If not installed yet, see the [Installation Guides](/getting-started/setup/installation-guides).
+- A Business Plan. For more information, see [Pricing](https://www.appsmith.com/pricing).
+- Access to update the role configuration.
+
+## Configure permissions
+
+Follow these steps to configure permissions:
+
+1. Navigate to Admin Settings by clicking the gear (⚙️) icon in the top right corner of your Appsmith Dashboard.
+2. From the sidebar, under **Access Control**, select **Roles**.
+3. Toggle **Default Roles** to show all the default roles.
+
+   <ZoomImage src="/img/GAC-Default-Roles-toggle.png" alt="Toggle the Default roles options to view Default Roles" caption="Toggle the Default roles options to view Default Roles"
+   />
+
+4. Click the pencil icon next to _Default Role For All Users_ that opens a configuration screen.
```

Comment:
They don't need to click the pencil icon, they can click on the name as well. The pencil icon is not a button, just a tag.